wafer/tickets/templates/wafer.tickets/claim.html

```diff
@@ -2,9 +2,9 @@
 {% load i18n %}
 {% load crispy_forms_tags %}
 {% block content %}
-{% if not WAFER_REGISTRATION_OPEN %}
-  {% trans "Registration is not currently open" %}
-{% elif not can_register %}
+{% if WAFER_REGISTRATION_OPEN != True %}
```

Comment:
If WAFER_REGISTRATION_OPEN is no longer going to be a simple boolean flag, we should rename it to something more descriptive. Something like WAFER_REGISTRATION_STAGE or WAFER_REGISTRATION_MODE, perhaps.